doc/deployment_guide.md

```diff
@@ -49,5422 +49,6234 @@ Red Hat®, Red Hat Enterprise Linux®, and Ceph are trademarks or registered tra
+* Added support for 100GbE networking with Mellanox ConnectX-5 network interface cards.
+* Added support for Red Hat Ceph Storage to use NVMe storage for OSD/Journal.
+* Enhancement of Huge Pages with JetPack CSP profile.
+* Enhancement of NUMA/CPU Pinning with JetPack CSP profile.
+* Added support of OVS-DPDK for 100G networking with Mellanox network interface cards (NICs).
+* Added support of SR-IOV offload for 100G networking with Connect X-5 Mellanox network interface cards (NICs) (Tech Preview in RedHat)
+* Added support of SR-IOV for 25GbE networking with Intel XXV710 network interface cards.
+* Added support for Cascade Lake 6230 processors
+* Added support for Distributed Virtual Router, (DVR).
+* Added support for auto-generation of .ini and .properties files for use in automated deployments.
+* Added support for VLAN aware VM.
+* Added support of Unity with Manila and Cinder
+* Added support of Octavia with Jetpack CSP profile
+* Added support of Barbican with Jetpack CSP profile
+* Added support of Satellite with Jetpack CSP profile
+* Added support for Bluestore
+
+
+## Before
+
+> Note: This guide assumes that you have racked the servers and networking hardware, and completed power and network cabling, as per the *Dell EMC Ready Architecture_for_Red_Hat OpenStack Platform Architecture Guide – Version 13.*
+
+The high-level steps required to install the Dell EMC Ready Architecture for Red Hat OpenStack Platform v13.2 using the automated installation procedures include:
+
+1.  Ensure that your environment meets the [Prerequisites](#Prerequisites)
+2.  Ensure that the [Dependencies](#Dependencies) are met.
+3.  *Determining Pool IDs*.
+4.  [Downloading and Extracting Automation Files]("https://github.com/dsp-jetpack/JetPack/").
+5.  [Preparing the Solution Admin Host Deployment](#Preparing-the-Solution-Admin-Host-Deployment).
+6.  [Deploying the SAH Node](#Deploying-the-SAH-Node).
+7.  [Deploying the Undercloud and the OpenStack Cluster](#Deploying-the-Undercloud-and-the-OpenStack-Cluster).
+
+## Prerequisites
+The following prerequisites must be satisfied before proceeding with a Dell EMC Ready Architecture for Red Hat OpenStack platform v13.2 deployment:
+
+> Note: All nodes in the same roles must be of the same server models, with identical HDD, RAM, and NIC configurations. So, all controller nodes must be identical to each other; all compute nodes must be identical to each other.
+
+* Hardware racked and wired per the [Dell EMC Ready Architecture for Red Hat OpenStack Platform Architecture Guide Version 13.1](https://www.dellemc.com/resources/en-us/asset/technical-guides-support-information/solutions/dell_emc_red_hat_ready_architecture_guide_v13.1.pdf).
+* Hardware configured as per the [Dell EMC Ready Architecture for Red Hat OpenStack Platform Architecture Guide Version 13.1](https://www.dellemc.com/resources/en-us/asset/technical-guides-support-information/solutions/dell_emc_red_hat_ready_architecture_guide_v13.1.pdf).
```

Comment:
13.**2**

Reply:
Kurt and Stretch are working on these guides for 13.2. I will update these links later.